docs/documentation-styleguide.md

```diff
@@ -0,0 +1,185 @@
+
+There is a [template](../rfcs/template.md) to guide you when writing an RFC. You can add other sections if appropriate. Use hyphens to separate words in the file name and save it as a `.md` file in the `carbon/rfcs/text` directory. If you need to add supporting images, save them in the `carbon/rfcs/images` directory.
+
+#### README files
+We write README files in MD format to tell collaborators about what Carbon and [Cypress.io](https://www.cypress.io) are, how collaborators can get started and where they can find further information or assistance.
+
+The [Carbon README](../README.md) contains:
+- a link to a more detailed 'Getting Started' guide;
+- a link to the [contributing guide](../CONTRIBUTING.md) explaing how to contribute to the project; and
+- license information.
+
+The [Cypress README](../cypress/README.md) contains:
+- instructions of how to install and run Cypress;
+- how to access test results; and
+- a link to the [testing styleguide](testing-styleguide.md) where a detailed explanation of how to write Cypress tests can be found.
+
+The [RFCs README](../rfcs/README.md) provides detailed instructions of when and how to write an RFC. 
+
+All README files are accessible through GitHub from their respective directories.
+
+When writing a README file, consider:
+- who your audience is;
+- what useful information you want in there;
+- if a table of contents is required;
+- the [use of links](#use-of-links); and
+- the [use of images](#use-of-images).
+
+Use hyphens to separate words in the file name and save it as a `.md` file in the relevant directory. If you need to add supporting images, save them in an `/images` directory within the same directory where you wish to store the README file. 
+
+#### Setup and Getting Started Guides
+Similar to a README, these guides are written in MD or MDX format. They contain more detailed information about how collaborators set up their [development environment](dev-environment-setup.md) and [get started](getting-started.stories.mdx) with developing their application using Carbon. These documents are stored in the `/docs` directory and accessible from GitHub.
+
+#### Styleguides
+We write various styleguides to provide some consistency for collaborators to the Carbon project to follow. This helps to maintain a standard way of doing things so we have a limited set of methods, tools and styles adopted throughout the repository. They often contain ordered and concise instructions so users can follow and repeat them easily. This is particularly useful for people new to the project who seek some common guidelines.
+
+Styleguides are stored in the `/docs` directory. It is a living directory so it continuously changes, with new guides added and others amended or removed. 
+
+When writing a styleguide, consider:
+- who your audience is and what prior understanding they have;
+- if a glossary is required;
+- formatting of [code in text](#code-in-text);
+- the [use of links](#use-of-links); and
```

Comment:
```suggestion
- the [use of links](#use-of-links);
```

Reply:
I followed the first example in "A list of very short points" in this guide for bullet point lists: http://www.plainenglish.co.uk/punctuating-bulleted-lists.html. However, if you think the second example without an "and" before the last point, then I'm happy to change.

Reply:
Removed the "and" in all lists in this document.